src/components/panel/side-panel.tsx

```diff
@@ -0,0 +1,34 @@
+import { useBuilder } from "../../context-providers/builder/hook";
+import { BasePanelHeader } from "./side-panel-header";
+import { ContentWrapper, Wrapper } from "./side-panel.styles";
+import { Toolbar } from "./toolbar";
+import { IBasePanelProps } from "./types";
+
+export const BasePanel = ({
```

Comment:
since the file has been renamed as `side-panel`, can rename this component as `SidePanel`